### PR TITLE
E_CLASSROOM-248 [UI] Quizzes List Page

### PR DESCRIPTION
**Pull request by catampongan**
### Issue Link
https://framgiaph.backlog.com/view/E_CLASSROOM-248

### Definition of Done
- [x] Markup is the same with the figma design
- [x] Columns are correct
- [x] Display dummy data
- [x] New path added to the sidebar
- [x] Search and filter components are present

### Commands to Run
- N/A

#### Main note
- N/A
#### Pages Affected
- http://localhost:3003/admin/quizzes

### Scenarios/ Test Cases
- [x] Markup is the same with the figma design
- [x] Columns are correct
- [x] Display dummy data
- [x] New path added to the sidebar
- [x] Search and filter components are present
#### User Interface
- N/A
#### User Experience 
- N/A
#### Functionality
- N/A
### Screenshots
image

### CODE REVIEW

**Comment by Teraaay** [comment]
### Working
- [x] Markup is the same with the figma design
- [x] Columns are correct
- [x] Able to display a dummy data
- [x] Search and filter components are present

### Comment/s
- [x] You forgot to add a Quizzes nav link in the admin sidebar. 
       - When done adding please link this page.

Please refer from Figma where to place the Quizzes nav link.

Figma Link: https://www.figma.com/file/0poKrCkzCV8RuL4eZTsDZ0/QUIZ-APP?node-id=3727%3A1125
     
### Screenshot/s
image

------
- Total Test Time: 0.25 hr

**Comment by Teraaay** [comment]
### Approved PR #100
- [x] Markup is the same as the Figma design
- [x] Columns are correct
- [x] Able to display a dummy data
- [x] Search and filter components are present
- [x] New path was added to the admin sidebar and linked to the page. 

### Screenshot/s
> image

----
- Total Test time: 0.5 hr
- Total Iterations: 1
- Total Comments: 2

File: src/pages/admin/quizzes/QuizList/index.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  ];
+
+  const renderList = () => {
+    return quizzes.map((quiz, idx) => {
+      return (
+        <tr key={idx}>
+          <td id={style.tBodyStyle}>{quiz.id}</td>
+          <td id={style.tBodyStyle}>{quiz.category}</td>
+          <td id={style.tBodyStyle}>{quiz.name}</td>
+          <td id={style.tBodyStyle1}>
```
Comment (karlolee50) [inline src/pages/admin/quizzes/QuizList/index.js]: change style name into more descriptive . maybe "button-column"

File: src/pages/admin/quizzes/QuizList/index.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+              <Button className={style.button}>Add a Quiz</Button>
+              <div>
+                <Table className={style.formatTable}>
+                  <thead>
+                    <tr>
+                      <td className={style.firstCol}>ID</td>
+                      <td className={style.firstCol}>Category</td>
+                      <td className={style.firstCol}>Name</td>
+                      <td className={style.firstCol1}>Edit</td>
+                      <td className={style.firstCol1}>Delete</td>
```
Comment (karlolee50) [inline src/pages/admin/quizzes/QuizList/index.js]: change class names as well

**Review by karlolee50 — CHANGES_REQUESTED** [review]
Review time: 15 minutes
Comment/s: 2

**Review by karlolee50 — APPROVED** [review]

**Pull request merged**